src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -930,6 +930,20 @@ def has_bluestore_label(device_path):
 
     return isBluestore
 
+def has_seastore_signature(device_path: str) -> bool:
+    magic = bytes.fromhex("67 45 8b 6b")
```

Comment:
We would probably want to decode a real signature once Seastore supports it later on, right?

Reply:
It would probably be better to support signatures in out implementation to avoid future issues.
Same as bluestore:
```
if signature.decode('ascii', 'replace') == bluestoreDiskSignature
```
Let's follow up on this after this PR is merged.